chia/cmds/farm_funcs.py

```diff
@@ -218,12 +218,17 @@ async def summary(rpc_port: int, wallet_rpc_port: int, harvester_rpc_port: int,
 
         print("Total size of plots: ", end="")
         plots_space_human_readable = total_plot_size / 1024 ** 3
-        if plots_space_human_readable >= 1024 ** 2:
-            plots_space_human_readable = plots_space_human_readable / (1024 ** 2)
-            print(f"{plots_space_human_readable:.3f} PiB")
-        elif plots_space_human_readable >= 1024:
+        if plots_space_human_readable >= 1024:
```

Comment:
I know we probably do not expect having `ZiB` here - but since now there are 3 conditional statements that are mostly completely the same code having some loop with division and unit mapping would be more reasonable?

Reply:
@jakub-polak I agree with you. It would be better to add a function like the one below and use it wherever we need to convert bytes to a human readable string.

```python
def get_human_readable_bytes_string(bytes: int) -> str:
    base = 1024
    bytes_float = bytes / base ** 2
    labels = {0 : 'MiB', 1 : 'GiB', 2 : 'TiB', 3 : 'PiB', 4 : 'EiB', 5 : 'ZiB', 6 : 'YiB'}
    n = 0
    while bytes_float >= base and n < 6:
        bytes_float /= base
        n += 1
    return f"{bytes_float:.3f} {labels[n]}"
```

Maybe one of the maintainers can recommend a suitable place for this function.